src/modules/displayComments.js

```diff
@@ -0,0 +1,71 @@
+import { BASE_URL } from './utils.js';
+import commentCounter from './commentCounter.js';
+import addComment from './addComment.js';
+import displayDetails from './displayDetails.js';
+// import { addLike } from './likes.js';
+let itemID = 0;
+const detailTitle = document.querySelector('.detail-title');
+const detailYear = document.querySelector('.detail-year');
+const detailDirector = document.querySelector('.detail-director');
+const detailStory = document.querySelector('.detail-story');
+const movieImg = document.querySelector('.movie-poster');
+const popup = document.querySelector('.popup');
+const getComments = async (i) => {
+  const response = await fetch(`${BASE_URL}/comments?item_id=item${i}`);
+  const data = await response.json();
+  return data;
+};
+const displayComments = async () => {
+  const popupSection = document.querySelector('.popup');
+  if (!popupSection.getElementsByTagName('h2').length) {
+    popupSection.innerHTML = `<i class="fa-solid fa-xmark fa-xl"></i>${popupSection.innerHTML}`;
+    const h2 = document.createElement('h2');
+    h2.id = 'comments';
+    const detailSec = document.querySelector('.details');
+    detailSec.after(h2);
+  }
+  document.addEventListener('click', async (event) => {
+    if (event.target.classList.contains('fa-xmark')) {
+      // detailTitle.innerHTML = '';
+      // detailYear.innerHTML = '';
+      // detailDirector.innerHTML = '';
+      // detailStory.innerHTML = '';
+      // movieImg.src = '';
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)